src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,246 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
 		)
 	);
 
+	$debug_info['buddypress-constants'] = array(
+		'label'       => __( 'BuddyPress Constants', 'buddypress' ),
+		'description' => __( 'These settings can alter where and how parts of BuddyPress are loaded or works.', 'buddypress' ),
+		'fields'      => array(
+			'BP_VERSION' => array(
+				'label' => 'BP_VERSION',
+				'value' => BP_VERSION,
+			),
+			'BP_DB_VERSION' => array(
+				'label' => 'BP_DB_VERSION',
+				'value' => BP_DB_VERSION,
+			),
+			'BP_REQUIRED_PHP_VERSION' => array(
+				'label' => 'BP_REQUIRED_PHP_VERSION',
+				'value' => BP_REQUIRED_PHP_VERSION,
+			),
+			'BP_PLUGIN_DIR' => array(
+				'label' => 'BP_PLUGIN_DIR',
+				'value' => BP_PLUGIN_DIR,
+			),
+			'BP_PLUGIN_URL' => array(
+				'label' => 'BP_PLUGIN_URL',
+				'value' => BP_PLUGIN_URL,
+			),
+			'BP_LOAD_SOURCE' => array(
```

Comment:
I wonder what's this one's goal. I'll need to check 🤔

Reply:
So the goal of this constant `BP_LOAD_SOURCE` is to force BuddyPress to load using the `src` subdirectory instead of the `build` directory when it exists. This is a contributor/developer need we shouldn't include into this panel. Moreover as there's only one check in `/bp-loader.php` it's not included into the `build` directory.